bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/validation/ConfigValidationException.java

```diff
@@ -50,11 +53,12 @@
      *
      * @throws NullPointException if given bundle or configuration description validation messages are null
      */
-    public ConfigValidationException(Bundle bundle, Collection<ConfigValidationMessage> configValidationMessages) {
-        Preconditions.checkNotNull(bundle, "Bundle must not be null");
+    public ConfigValidationException(TranslationProvider translationProvider,
```

Comment:
This class is instantiated by tests and ConfigDescriptionValidatorImpl.
ConfigDescriptionValidatorImpl itself is a component and so it can access the bundle without using `FrameworkUtil`.
We could keep the bundle as an argument and there is no use to introduce the usage of `FrameworkUtil` here just to drop it later.

Reply:
Good point, thanks!